lib/install/config/webpack/paths.yml

```diff
@@ -0,0 +1,7 @@
+# Restart webpack-watcher or webpack-dev-server if you make changes here
+paths:
+  src_path: app/javascript
+  config_path: config/webpack
+  node_modules_path: node_modules
+  dist_dir: packs
+  dist_path: public/packs
```

Comment:
Or, perhaps we don't use `paths` key and use `webpack` or `webpacker` because we can use this config file to setup other things going forward like - extensions etc. Also, then having `dist_dir` etc. wouldn't feel odd.